controllers/app.controllers.js

```diff
@@ -0,0 +1,20 @@
+const { fetchTopics } = require('../models/index')
+
+exports.getTopics = (req,res,next)=>{
+    const validParams = [];
+    const queryKeys = Object.keys(req.query);
+
+    const invalidParam = queryKeys.some(key=> !validParams.includes(key))
```

Comment:
Interesting way of doing this, not quite sure what query will be added to the topics though